src/EmailCardList/EmailCardList.js

```diff
@@ -7,16 +9,19 @@ const EmailCardList = (props) => {
         return dateObject.toLocaleDateString("en-GB")
     }
     const emailItems = props.allEmailSnippets;
-    const emailCardsArray = emailItems.map((emailItem) =>
-        <EmailCard name={emailItem.name} date_created={formatDateForDisplay(emailItem.date_created)}
-                   subject={emailItem.subject} body={emailItem.body} read={emailItem.read}/>
+    const emailCardsArray = emailItems.map((emailItem) => 
+        <EmailCard id={emailItem.id} name={emailItem.name} date_created={formatDateForDisplay(emailItem.date_created)}
```

Comment:
rename id to emailId so that it is clear that it's a meaningful prop rather than an id attribute in an HTML sense